docs/source/api.rst

```diff
@@ -53,6 +53,15 @@ The information returned for each entity is like the following::
       ]
     }
 
+""""""""""""""""""""""""""""""
+GET /api/agency/query='privacy'
```

Comment:
I think this should be `api/agency/?query=privacy` the documentation query doesn't work.